regression/dataset_mixing.yaml

```diff
@@ -0,0 +1,79 @@
+name: streaming-regression-test-deterministic-shuffling
+compute:
+  gpus: 8 # Number of GPUs to use
+  # cluster: TODO # Name of the cluster to use for this run
+command: |-
+  pip uninstall -y mosaicml-streaming
+  cd streaming
+  pip install -e '.[dev]'
+
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_0/ --num-samples 5000 --size_limit 4096 --seed 0
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_1/ --num-samples 5000 --size_limit 4096 --seed 1
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_2/ --num-samples 5000 --size_limit 4096 --seed 2
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_3/ --num-samples 5000 --size_limit 4096 --seed 3
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_4/ --num-samples 5000 --size_limit 4096 --seed 4
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_5/ --num-samples 5000 --size_limit 4096 --seed 5
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_6/ --num-samples 5000 --size_limit 4096 --seed 6
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_7/ --num-samples 5000 --size_limit 4096 --seed 7
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_8/ --num-samples 5000 --size_limit 4096 --seed 8
+  python regression/synthetic_dataset.py --create --name numberandsaydataset --out /tmp/streaming_dataset/stream_9/ --num-samples 5000 --size_limit 4096 --seed 9
+
+  # Check sample order with stream choose
+  composer -n 1 regression/iterate_data.py --num_canonical_nodes 128 --batch_size 32 --sample-order-file /tmp/test_samples_0.txt
```

Comment:
@karan6181 I'm getting a lint error for too long of a line, how do I split across multiple lines?